src/test/java/com/afesguerra/sonarqube/plugin/sns/sns/AmazonSNSClientProxyTest.java

```diff
@@ -1,7 +1,5 @@
-package com.afesguerra.sonarqube.plugin.sns;
+package com.afesguerra.sonarqube.plugin.sns.sns;
```

Comment:
?? `sns.sns` ??

Reply:
so, the actual SNS client for the Sonarqube SNS plugin... kiiiiinda makes sense (+ couldn't think of a better package path)